sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -77,31 +79,32 @@ class MicroBatchExecution(
       sparkSession.sqlContext.conf.disabledV2StreamingMicroBatchReaders.split(",")
 
     val _logicalPlan = analyzedPlan.transform {
-      case streamingRelation@StreamingRelation(dataSourceV1, sourceName, output) =>
-        toExecutionRelationMap.getOrElseUpdate(streamingRelation, {
+      case s @ StreamingRelation(dsV1, sourceName, output) =>
```

Comment:
These renames are cosmetic, and unnecessary changes like this cause commit conflicts. I think these changes should be reverted.

Reply:
This is to make the naming consistent, see https://github.com/apache/spark/pull/20647/files#diff-c2959c723f334c32806217216014362eL89

In general we don't forbid users to fix some code style issue in related PRs, otherwise we need to have code-style-fix-only PRs, which is not the common case.

Reply:
If you are touching that specific code then its fine to fix the style, but in general I tend to agree that it makes the diff harder to read and commit harder to back port if you include spurious changes.

I've even seen guidelines that specifically prohibit fixing style just to fix style since it obfuscates the history.

Reply:
yea I was touching the code here to retain the `DataSourceV2` instance, and fixed the code style.

Reply:
I don't think this counts as touching the code. This is near code you needed to change, but it makes the changes in this patch larger and more likely to conflict.

Changes like this make maintenance much more difficult because they can cause conflicts that weren't necessary. After this change, commits would use `s` instead of `streamingRelation`, which would prevent any commit after this to be applied to a branch without this one. If that is avoidable because the change is not necessary, I want to avoid it.